Support the `hdbcli` client library (#10595)
DataDog/integrations-core: sap_hana/README.md

   GRANT DD_MONITOR TO <USER>;
   ```

#### Dependencies

> The [hdbcli][10] client library is supported but must be installed manually for licensing reasons. This is the recommended way to interface with SAP HANA since it is the only officially maintained library.

Reviewer: is it required?

Author: No, I'll make that more explicit 👍

   GRANT DD_MONITOR TO <USER>;
   ```

#### Dependencies

> Although not required, it is recommended that the [hdbcli][10] client library be installed since it is the only officially maintained library and supports many new features.

Reviewer 2: ```suggestion
Although not required, it is recommended to install the [hdbcli][10] client library since it is the only officially maintained library and supports many new features.
```
